pkg/cmd/create/create.go

```diff
@@ -32,33 +26,14 @@ func NewCmdCreate() *cobra.Command {
 				return fmt.Errorf("unable to create script dir: %v", err)
 			}
 
-			return create(script, args)
+			return storage.Create(script)
```

Comment:
`storage.Create` sounds like you're creating a new storage.

Maybe `storage.AddScript(script)` will be better.